src/styles/utils/keyframes.scss

```diff
@@ -0,0 +1,8 @@
+@keyframes rotate {
+  from {
+    transform: rotate(0);
```

Comment:
Your minutes and hour hands don't center, because you don't take into account the width of the arrows during animation. 
<img width="801" alt="Screenshot 2023-06-02 at 16 57 29" src="https://github.com/mate-academy/layout_stop-watch/assets/28480057/c725420d-9d8f-41d4-a6fa-4926b2f66e9a">

When you set `transform: translate(-50%, -50%);` in mixin `absolutePositioning` **without** animation - everything looks and works well, but when you add animation you rewrite transform property of mixin with keyframes values. So we have that bug.